source/includes/harvest/_jobs.md

```diff
@@ -257,15 +263,20 @@ curl 'https://harvest.greenhouse.io/v1/jobs'
         "status": "closed",
         "opened_at": "2015-11-19T19:53:32.564Z",
         "closed_at": "2016-01-26T23:59:07.592Z",
-        "application_id": 24709881
+        "application_id": 24709881,
```

Comment:
@tdphillipsjr This application ID is the hired application for that opening, right?

Reply:
Yes, this is defined near your previous comment (Line 165 - https://github.com/grnhse/greenhouse-api-docs/pull/143/files#diff-24365b61af44ab1df83d874522206c4dR165)

Reply:
Hoo boy I was out of it last night.

Reply:
👍